ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource

```diff
@@ -119,3 +125,9 @@ Delete Pipeline Server
         Sleep  1s
     END
     Should Be True    ${pod_count} == 0    Pods weren't removed
+
+Verify There Is No "Error Displaying Pipelines" After Creating Pipeline Server
+    [Documentation]    Verify me message "Error displaying Pipelines" after creating pipeline server
+    ...    Bug: RHOAIENG-2931
+    Run Keyword And Ignore Error       Wait Until Page Contains    Error displaying pipelines    timeout=60s
+    Run Keyword And Warn On Failure    Page Should Not Contain     Error displaying pipelines
```

Comment:
Just to be sure - at the beginning there is an `Error Displaying Pipelines` message and we give it 60 seconds to disappear. This is the intention, correct?

Reply:
The "Error Displaying Pipelines" message can take a few secs to appear, so we wait for it for 60s. After that, if the error message is not there we do nothing  (bug is fixed) but if it's present we log a Warning (I was hesitating if to fail the test, maybe I'll do that in Sanity)